app/boards/arm/nice_nano/nice_nano.dtsi

```diff
@@ -50,6 +50,13 @@
     pinctrl-names = "default", "sleep";
 };
 
+&spi0 {
```

Comment:
Oh, can we please use `spi1`?

nRF52 doesn't allow using the same numbered SPI & i2c peripheral at the same time. So someone trying to use both would have problems if we stuck with this.

Reply:
Looks like `nice_nano`, `nrfmicro`, `mikoto`, `bluemicro840` all use `pro_micro_spi: spi0`. I think it makes sense to use `pro_micro_spi` for these since they are standard Pro Micro pin locations so it might be nice to rename them to `spi1`.

I suppose (maybe separately) it would be nice to define pinctrl for others as well.

Reply:
@caksoylar - I can make these changes. I didn't realize that the other boards didn't have SPI defined as well.

So, I'm reading that I should:
* update the commit message to convention
* change pro_micro_spi for all nrf boards to be spi1
* change the definitions I made in this commit to be spi1 instead of spi0